### DISCUSSION TITLE
Starting AIS Catcher on Mac

### DISCUSSION THREAD

**Discussion by Joe295398** [jvde-github/AIS-catcher#177] [Q&A] [answered]
I have successfully gotten AIS Catcher to work on Windows as well as a Raspberry PI, but I can not figure out how to get it to work on MacOS.  Any help would be appreciated, thank you.

**Comment by jvde-github** [comment]
Did you manage to install brew? Any further insights on errors you encounter?

**Reply by Joe295398** [reply]
Yes brew is installed, dependencies are in place and I am to the install AIS Catcher section. Coping the install code into terminal nets me this:

joeg@Joes-MBP-16 AIS-catcher % git clone https://github.com/jvde-github/AIS-catcher.git
cd AIS-catcher
mkdir build
cd build
cmake ..
make
sudo make install
Cloning into 'AIS-catcher'...
remote: Enumerating objects: 17180, done.
remote: Counting objects: 100% (1389/1389), done.
remote: Compressing objects: 100% (285/285), done.
remote: Total 17180 (delta 1132), reused 1333 (delta 1098), pack-reused 15791
Receiving objects: 100% (17180/17180), 92.54 MiB | 5.52 MiB/s, done.
Resolving deltas: 100% (11348/11348), done.
zsh: command not found: cmake
make: *** No targets specified and no makefile found.  Stop.
Password:

**Reply by jvde-github** [reply, marked as answer]
Looks like you do not have cmake installed. I am traveling so cannot access my mac. Assume it is 'brew install cmake', might also be some other dependencies later on.

**Reply by Joe295398** [reply]
Okay yes, that was it. Thank you!

**Reply by jvde-github** [reply]
Glad that it worked. I also now notice I am a bit light on mac installation instructions. Perhaps some day will fix it. Enjoy the program.

**Reply by splobsterman** [reply]
Did you ever have a chance to add to the Mac install instructions?

**Reply by jvde-github** [reply]
I added something here: https://github.com/jvde-github/AIS-catcher/blob/main/README.md#ubuntu-raspberry-pi-macos-msvc

Do you think that is sufficient. You need to install brew, some basic dependencies and then the tools for your specific hardware (e.g. rtlsdr)

**Comment by splobsterman** [comment]
Thank you, will look. I don't use Brew, having been a MacPorts user for ages but I should be able to work it out. I'll try to let you know!

**Reply by jvde-github** [reply]
I don’t have any experience with macports but you could give it a try, installing git, cmake and rtl-sdr library. If it compiles should be fine

**Comment by splobsterman** [comment]
Turns out I had those pre-req's in place via Macports and it built fine once I sorted out the build instructions. Using MacOS Monterey (12.7.5) on an M1 Mac mini. Using the now default terminal shell zsh.
